packages/transport/src/modules/sync/SyncController.ts

```diff
@@ -410,14 +411,15 @@ export class SyncController extends TransportController {
         const changedItems = new ChangedItems();
 
         for (const externalEvent of externalEvents) {
+            const externalEventObject = BackboneExternalEvent.fromAny(externalEvent);
```

Comment:
Does this require a `try {} catch {}` block, as below?

Reply:
It really does. in case of a deserialization error we have to send the backbone that the external event could not be processed.

Reply:
Is there already an API to do that ?

Reply:
see below in the catch block

Reply:
ah ok than i would just move the parsing in the try catch below